floris/tools/optimization/scipy/yaw_wind_rose_parallel.py

```diff
@@ -513,7 +515,10 @@ def optimize(self):
         with MPIPoolExecutor() as executor:
             if self.ti is None:
                 for df_opt_one in executor.map(
-                    self._optimize_one_case, self.ws.values, self.wd.values
+                    self._optimize_one_case,
+                    self.ws.values,
+                    self.wd.values,
+                    self.df_base.power_baseline.values,
```

Comment:
the super init is already calling self._get_initial_farm_power(), so maybe it makes sense for the parallel class to use the initial power values that were calculated there for consistency.

Reply:
After thinking about it, if this were the case the parallel class would not be able to take advantage of parallelizing when get_initial_farm_power() is called in the parent class.